app/controllers/api/v1/users/confirmations_controller.rb

```diff
@@ -0,0 +1,28 @@
+class Api::V1::Users::ConfirmationsController < Devise::ConfirmationsController
+  # GET /resource/confirmation/new
+  # def new
+  #   super
+  # end
+
+  # POST /resource/confirmation
+  # def create
+  #   super
+  # end
+
+  # GET /resource/confirmation?confirmation_token=abcdef
+  # def show
+  #   super
+  # end
+
+  # protected
+
+  # The path used after resending confirmation instructions.
+  # def after_resending_confirmation_instructions_path_for(resource_name)
+  #   super(resource_name)
+  # end
+
+  # The path used after confirmation.
+  # def after_confirmation_path_for(resource_name, resource)
+  #   super(resource_name, resource)
+  # end
+end
```

Comment:
- [ ] Just like the previous TSE suggested, I want to suggest it too, I believe you could delete these commented pieces of code, since they aren't really used for anything. I would take a look at all of your files and make sure there's none.